projects/element-ng/resize-observer/si-resize-observer.directive.ts

```diff
@@ -23,6 +23,8 @@ import { ElementDimensions, ResizeObserverService } from './resize-observer.serv
 export class SiResizeObserverDirective implements OnInit, OnDestroy {
   /** @defaultValue 100 */
   readonly resizeThrottle = input(100);
+  /** @defaultValue true */
+  readonly emitInitial = input(true);
```

Comment:
Needs attribute converter and I think it makes sense to change the default with v48 to false.


So please add a TODO comment unless you see a reason why the default should be true.

Reply:
@spike-rabbit Sorry, but this is just wrong! Having an attribute converter and then changing the default to false makes absolutely no sense at all. The default MUST remain true! Otherwise it will break almost all consumers. Doing this just so that boolean transform can be used is just pure evil.